server_admin/topics/sso-protocols/oidc.adoc

```diff
@@ -97,6 +97,80 @@ This is used by clients running on internet-connected devices that have limited
+
+An administrator can configure Client Initiated Backchannel Authentication (CIBA) related operations as `CIBA Policy` per realm.
+
+====== CIBA Policy
+
+An administrator carries out the following operations on the `Admin Console` :
+
+- Open the `Authentication -> CIBA Policy` tab.
+- Configure items and click `Save`.
+
+The configurable items and their description follow.
+
+|===
+|Configuration|Description
+
+|Backchannel Token Delivery Mode
+|Specifying how the CD (Consumption Device) gets the authentication result and related tokens. There are three modes, "poll", "ping" and "push". {project_name} only supports "poll". The default setting is "poll". This configuration is required.
+ For more details, see https://openid.net/specs/openid-client-initiated-backchannel-authentication-core-1_0.html#rfc.section.5[CIBA Specification].
+
+|Expires In
+|The expiration time of the "auth_req_id" in seconds since the authentication request was received. The default setting is 120. This configuration is required.
+ For more details, see https://openid.net/specs/openid-client-initiated-backchannel-authentication-core-1_0.html#successful_authentication_request_acknowdlegment[CIBA Specification].
+
+|Interval
+|The interval in seconds the CD (Consumption Device) needs to wait for between polling requests to the token endpoint. The default setting is 5. This configuration is optional.
+ For more details, see https://openid.net/specs/openid-client-initiated-backchannel-authentication-core-1_0.html#successful_authentication_request_acknowdlegment[CIBA Specification].
+
+|Authentication Requested User Hint
+|The way of identifying the end-user for whom authentication is being requested. The default setting is "login_hint".  There are three modes, "login_hint", "login_hint_token" and "id_token_hint". {project_name} only supports "login_hint". This configuration is required.
+ For more details, see https://openid.net/specs/openid-client-initiated-backchannel-authentication-core-1_0.html#rfc.section.7.1[CIBA Specification].
+
+|===
+
+====== Provider Setting
+
+The CIBA grant uses the following two providers.
+
+. Authentication Channel Provider : provides the communication between {project_name} and the entity that actually authenticates the user via AD (Authentication Device).
+. User Resolver Provider : get `UserModel` of {project_name} from the information provided by the client to identify the user.
+
+{project_name} has both default providers. However, the administrator needs to set up Authentication Channel Provider like this:
+
+[source,xml]
+----
+<spi name="ciba-auth-channel">
+    <default-provider>ciba-http-auth-channel</default-provider>
+    <provider name="ciba-http-auth-channel" enabled="true">
+        <properties>
+            <property name="httpAuthenticationChannelUri" value="http://backend.internal.example.com/auth"/>
```

Comment:
Is it possible to rather use "https" for the sake of better security? Like `https://backend.internal.example.com/auth` ?

Reply:
Yes, that's right. I will change it to `https://backend.internal.example.com/auth` .

Reply:
+1, Thanks!